packages/web-components/src/global/color-mode.css

```diff
@@ -2434,22 +2437,27 @@ ic-theme,
     --ic-skip-link-background: var(--ic-color-background-primary);
 
     /* Status tags */
-    --ic-status-tag-text: var(--ic-color-text-inverse);
-    --ic-status-tag-text-inverted: var(--ic-color-text-inverse);
+    --ic-status-tag-text: var(--ic-color-text-primary-light);
```

Comment:
shouldn't `--ic-status-tag-text` and `--ic-status-tag-text-inverted` be the other way around in dark mode?

![image](https://github.com/user-attachments/assets/2aa9faf2-bb30-44d0-9935-57728fcafb7d)